_layouts/newsletter.html

```diff
@@ -2,15 +2,31 @@
 layout: default
 --- 
 <link rel="stylesheet" href="/assets/css/main.css">
+
+{% assign baseurl = page.permalink | remove_first: "/" | remove_first: page.lang %}
+
+{% for lang in site.languages %}
+  {% assign localization = "/" | append: lang | append:baseurl %}
+  {% assign locale = site.posts | where:"permalink", localization %}
+  {% assign localizations = localizations | concat: locale %}
+{% endfor %}
+
 <article class="newsletter">
   <header class="post-header">
+      {% if localizations.size > 0 %}
+        <a href="{{ '/en' | append:baseurl }}">en</a>
+        {% for locale in localizations %}
+          | <a href="{{ locale.url }}">{{locale.lang}}</a>
+        {% endfor %}
+      {% endif %}
```

Comment:
I think this list would look better right aligned, or maybe center aligned beneath the date; however, I'm ok with any style you like.

Reply:
I agree. Let's right-align this.

Reply:
Right aligned!